fossunited/www/newsletter/index.html

```diff
@@ -1,176 +1,560 @@
       </ul>
     </div>
   </div>
 </section>
-
-<section id="footer-cta">
-  <div id="cta-container" class="base-container container-center h-full">
-    <div id="cta">
-      <div class="text-center">
-        <h4>Connect with us</h4>
-        <p>Drop a message in our telegram group.</p>
-        <div>
-          <button type="submit" id="telegram-channel">
-            <a href="https://t.me/fossunited" alt="@fossunited telegram">Get in touch</a>
-          </button>
-        </div>
-      </div>
+<section id="footer">
+  <div class="base-container container-center m-auto">
+    <div id="cta-container" class="text-center">
+      <h4>Get in Touch</h4>
+      <p>
+        Write to us at foundation@fossunited.org or send us a message on the
+        forum. <br />For event sponsorship, write to us at
+        <a href="mailto:grants@fossunited.org">grants@fossunited.org</a>.
+        <br />Join our
+        <a href="https://t.me/fossunited" alt="@fossunited telegram">
+          Telegram Community</a
+        >.
+      </p>
     </div>
   </div>
 </section>
-
-<!-- Subject to display Change -->
 <section id="subscribed-page">
   <div id="success d-flex">
     <div id="success-wrapper">
       <div id="success-icon-div">
-        <svg class="success-icon container-center icon"><use href="#icon-solid-success"></use></svg>
+        <svg class="success-icon container-center icon">
+          <use href="#icon-solid-success"></use>
+        </svg>
       </div>
-      <div class="text-center ">
+      <div class="text-center">
         <p>
-        You are subscribed to our newsletter! You will be receiving updates every month about FOSS and FOSS United.
+          You are subscribed to our newsletter! You will be receiving updates
+          every month about FOSS and FOSS United.
         </p>
       </div>
     </div>
   </div>
 </section>
 
+<script src="/newsletter/index.js" type="text/javascript"></script>
 <style>
-#hero h1 {
-  font-size: 40px;
-}
-
-.news-icon img {
-  margin-bottom: -40px !important;
-}
-
-#hero p {
-  margin: auto;
-  width: auto;
-  max-width: 900px;
-  font-size: 20px;
-}
-
-.newsletter-icon img {
-  margin-bottom: -50px !important;
-}
-
-#newsletters-list {
-  margin: auto;
-}
-#email {
-  border: 1px solid black;
-  border-radius: 8px;
-  width: 360px;
-  height: 48px;
-  padding-left: 10px;
-}
-#news {
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  gap: 12px;
-  padding: 24px 48px;
-  background: var(--gray-50, #F9FAFB);
-  color: #08b74f;
-  margin: 15px;
-}
-
-    #news a {
-      color: #08b74f;
-      max-width: 500px;
-    }
-
-    p {
-      padding: 0px 20px 0px 20px;
-    }
-    #subscribe {
-      padding: 12px;
+  .newsletter-home-page {
+    display: inline-flex;
+    padding: 6.6875rem 20rem 0rem 20rem;
+    flex-direction: column;
+    justify-content: flex-end;
+    margin: auto;
+    align-items: center;
+    gap: 2.5rem;
+    height: auto;
+    background: #f5f5f5;
+  }
```

Comment:
container will make this easier, and for background you can give it a background color easily